doc/interfaces/denoise.rst

```diff
@@ -0,0 +1,90 @@
+=========
+Denoising
+=========
+
+This tutorial walks through the steps for denoising of diffusion-weighted MR images.
+There can be multiple denoising techniques.
+
+First you need to download the desired data.
+
+------------------------
+Downloading the Data Set
+------------------------
+
+You can view all the datasets using the command::
+
+    dipy_fetch list
+
+To fetch some specific dataset, you can use the following command::
+
+    dipy_fetch specific_dataset
+
+For example, to download sherbrooke_3shell dataset, you can use the following
+command::
+
+    dipy_fetch sherbrooke_3shell
+
+-------------------------
+Denoising using Local PCA
+-------------------------
+
+PCA-based denoising algorithms are effective denoising methods because they explore 
+the redundancy of the multi-dimensional information of diffusion-weighted datasets
+
+We are using isbi2013_2shell dataset. You can also use your own data.
+
+Following workflow requires input_files Path to the input volumes, bvalues_files Path
+to the bvalues files and bvectors_files Path to the bvectors files.
+
+Run the following workflow::
+
+    dipy_denoise_lpca /.dipy/isbi2013/phantom64.nii.gz /.dipy/isbi2013/phantom64.bval /.dipy/isbi2013/phantom64.bvec
+
+This workflow will generate a denoised volume (default: dwi_lpca.nii.gz) in the
+input directory by default.
```

Comment:
This command will denoise the diffusion image and save it to a file named `dwi_lpca.nii.gz` by default, located in the input directory also by default.

@areeshatariq Are you sure it saves it to the input directory by default? If yes, :+1:. Also, explain the option to specify where the output can be stored.

I would refer to the command lines as what they are. The `workflow` term is not accurate enough to me to refer to these.

Reply:
Addressed in  a79ccaf